app/src/main/java/com/cornellappdev/uplift/models/UpliftClass.kt

```diff
@@ -0,0 +1,20 @@
+package com.cornellappdev.uplift.models
+
+import androidx.compose.runtime.State
+import androidx.compose.runtime.mutableStateOf
+import java.util.*
+
```

Comment:
For future code quality, Ima implement a policy of adding comments at the top of each class/composable/etc explaining what this it

Reply:
so write headers for all of your data classes etc

Reply:
Addressed in commit [49de32b](https://github.com/cuappdev/uplift-android/pull/1/commits/49de32bb4aa554f03f64ba7dc66a5c4aa98851a9).